tools/test-framework/src/types/single/node.rs

```diff
@@ -238,6 +238,7 @@ impl FullNode {
             packet_filter: Default::default(),
             websocket_addr,
             rpc_addr,
+            cell_emitter_addr: Url::from_str("http://127.0.0.1").unwrap(), // set it as normal in test
```

Comment:
What about the port of `cell_emitter_addr`?

Reply:
this field in test-framework isn't necessary, so it can be everything including without port